Firestore/Source/Model/FSTFieldValue.mm

```diff
@@ -907,14 +931,22 @@ - (NSUInteger)hash {
   return [self.internalValue hash];
 }
 
-- (id)valueWithOptions:(FSTFieldValueOptions *)options {
+- (id)value {
   NSMutableArray *result = [NSMutableArray arrayWithCapacity:_internalValue.count];
   [self.internalValue enumerateObjectsUsingBlock:^(FSTFieldValue *obj, NSUInteger idx, BOOL *stop) {
     [result addObject:[obj value]];
```

Comment:
I'll write a test for this, but it was probably a bug (`options` weren't propagated to the elements in the array). Unless perhaps it's impossible to have `ServerTimestampValue`s in an array?

Reply:
It is indeed impossible to have server timestamp values in an array [because the protocol currently doesn't define how to reference an array element via a field path, so we can't create the necessary FieldTransform], but that should be rejected in the API layer (in FSTUserDataConverter probably)...  so it was probably a bug, although benign.

Reply:
Sounds like it would be hard/impossible to test?